### DISCUSSION TITLE
[Support] http-proxy 代理不支持文件上传吗？

### DISCUSSION THREAD

**Discussion by ilaziness** [midwayjs/midway#3961] [Q&A] [answered]
### Describe the problem（描述问题）

midway版本是3.12.0。
配置的httpProxy.strategy，测试post multipart/form-data上传文件到后端服务拿不到数据，是不支持代理文件上传吗？
找了下文档没有看到相关的东西。

### Midway Versions（Midway 版本）

>> Start to check your midway component version...

✓ @midwayjs/faas-typings(not installed)
✓ @midwayjs/fc-starter(not installed)
✓ @midwayjs/serverless-http-parser(not installed)
✓ @midwayjs/async-hooks-context-manager(3.14.4)
✓ @midwayjs/axios(3.14.4)
✓ @midwayjs/bootstrap(3.14.4)
✓ @midwayjs/bull(3.14.4)
✓ @midwayjs/bull-board(not installed)
✓ @midwayjs/cache-manager(3.14.4)
✓ @midwayjs/captcha(not installed)
✓ @midwayjs/casbin(not installed)
✓ @midwayjs/casbin-redis-adapter(not installed)
✓ @midwayjs/casbin-typeorm-adapter(not installed)
✓ @midwayjs/code-dye(not installed)
✓ @midwayjs/consul(not installed)
✓ @midwayjs/core(3.14.4)
✓ @midwayjs/cos(not installed)
✓ @midwayjs/cron(not installed)
✓ @midwayjs/cross-domain(3.14.4)
✓ @midwayjs/decorator(3.14.4)
✓ @midwayjs/etcd(not installed)
✓ @midwayjs/express-session(not installed)
✓ @midwayjs/faas(not installed)
✓ @midwayjs/grpc(not installed)
✓ @midwayjs/http-proxy(3.14.4)
✓ @midwayjs/i18n(3.14.4)
✓ @midwayjs/info(3.14.4)
✖ @midwayjs/jwt(current: 3.16.2, allow: ["3.14.4","3.14.7","3.14.10"])
✓ @midwayjs/kafka(not installed)
✓ @midwayjs/leoric(not installed)
✓ @midwayjs/mikro(not installed)
✓ @midwayjs/mock(3.14.4)
✓ @midwayjs/mongoose(not installed)
✓ @midwayjs/oss(not installed)
✓ @midwayjs/otel(not installed)
✓ @midwayjs/passport(not installed)
✓ @midwayjs/process-agent(not installed)
✓ @midwayjs/prometheus(not installed)
✓ @midwayjs/prometheus-socket-io(not installed)
✓ @midwayjs/rabbitmq(not installed)
✓ @midwayjs/redis(3.14.4)
✓ @midwayjs/security(not installed)
✓ @midwayjs/sequelize(not installed)
✓ @midwayjs/session(3.14.4)
✓ @midwayjs/socketio(not installed)
✓ @midwayjs/static-file(not installed)
✓ @midwayjs/swagger(3.14.6)
✓ @midwayjs/tablestore(not installed)
✓ @midwayjs/tags(not installed)
✓ @midwayjs/typegoose(not installed)
✓ @midwayjs/typeorm(not installed)
✖ @midwayjs/upload(current: 3.15.6, allow: ["3.14.4","3.14.7","3.14.10"])
✓ @midwayjs/validate(3.14.4)
✓ @midwayjs/version(not installed)
✓ @midwayjs/view(not installed)
✓ @midwayjs/view-ejs(not installed)
✓ @midwayjs/view-nunjucks(not installed)
✓ @midwayjs/web(not installed)
✓ @midwayjs/express(not installed)
✓ @midwayjs/koa(3.14.4)
✓ @midwayjs/ws(not installed)
************************************************************************************************************************
>> Check complete, found  2  problem.
>> Use -u to show update list that can be upgraded to the latest version.
>> Use -m to show updates that can be upgraded to the most compatible version.
>> Use -u -w or -m -w to write to the package.json file and update the lock file if it exists.
>> Please check the result above.
************************************************************************************************************************

**Comment by czy88840616** [comment, marked as answer]
不支持流式部分处理。

**Reply by ilaziness** [reply]
好
